tracer/test/Datadog.Trace.TestHelpers.AutoInstrumentation/InstrumentationVerification.cs

```diff
@@ -18,26 +18,30 @@ internal static class InstrumentationVerification
 {
     /// <summary>
     /// Configuration key for enabling or disabling the instrumentation verification.
```

Comment:
```suggestion
    /// Configuration key for enabling or disabling writing the instrumentation changes to disk (to allow for post-mortem instrumentation verification analysis).
```

Reply:
Fixed in d6e34e581c70cc2271806c12db2c3d4f2caf78f1